app/components/common/SectionHeader.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { FC } from 'react'
+import { View } from 'react-native'
+
+export interface IProps {
+  title: string
+}
+
+export const sectionHeaderHeight = 10
+
+const SectionHeader: FC<IProps> = function () {
```

Comment:
```suggestion
const SectionHeader: FC<IProps> = () => {
```suggestion
const SectionHeader: FC<IProps> = function () {
```
```

Reply:
done